frontend/src/components/pages/CampCreation/ScheduleSessions/index.tsx

```diff
@@ -1,10 +1,25 @@
-import React from "react";
 import { Box, Text } from "@chakra-ui/react";
+import React, { useState } from "react";
+import { CreateCampSession } from "../../../../types/CampsTypes";
+// import SessionsCalendar from "./SessionsCalendar";
+import SessionSidePanel from "../SessionSidePanel";
+
+const ScheduleSessionsPage = (): React.ReactElement => {
+  const [scheduledSessions, setScheduledSessions] = useState<
+    CreateCampSession[]
+  >([]);
 
-const ScheduleSessionsPage = (): JSX.Element => {
   return (
-    <Box>
+    <Box ml="230px" mt="50px">
       <Text textStyle="displayXLarge">Schedule Sessions</Text>
+      <Text textStyle="heading" mb={8}>
+        Waterloo Photography Camp 2022 @7:00 AM - 3:00PM
+      </Text>
+      {/* <SessionsCalendar sessions={scheduledSessions}/> */}
+      <SessionSidePanel
```

Comment:
I feel like it would make more sense to do this in a different PR when we put the status header, calendar, and side panel together. 

I can do it in this PR but I feel like this should focus more on just creating the side panel.